app/src/main/java/org/oppia/app/home/topiclist/PromotedStoryListAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.app.home.topiclist
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.databinding.PromotedStoryCardBinding
+
+class PromotedStoryListAdapter(private val itemList: MutableList<PromotedStoryViewModel>) :
```

Comment:
Please add KDoc to new class.

Reply:
Added this KDoc:

`/** Adapter to bind promoted stories to [RecyclerView] inside [HomeFragment] to create carousel. */`